.github/workflows/wheels-build.yaml

```
@@ -20,19 +20,23 @@ on:
         required: true
         type: string
       package-type:
+        description: "One of: [cpp, python]"
         required: false
         type: string
       wheel-name:
         required: false
         type: string
+        description: "The name of the wheel"
```

```suggestion
        description: "Distribution name, without any other qualifiers (e.g. 'pylibcudf', not 'pylibcudf-cu12-cp311-manylinux_2_24_aarch64')"
```

If I was already unsure what to fill in for `wheel-name`, I don't think this description as-is would be much help to me. Would you please consider this rewording?